consoles/video_stream.pm

```diff
@@ -61,6 +61,7 @@ sub disable ($self, @) {
 
 # uncoverable statement count:1..5 note:the function is redefined in tests
 sub _v4l2_ctl ($device, $cmd_prefix, $cmd) {
+    $cmd_prefix //= "";
     my @cmd = split(/ /, $cmd_prefix);    # uncoverable statement
```

Comment:
Since this is only used once:
```suggestion
    my @cmd = split(/ /, $cmd_prefix // '');    # uncoverable statement
```
We also prefer single quotes unless interpolation is used.